dotnet/src/Skills/Skills.Web/Bing/BingConnector.cs

```diff
@@ -23,14 +23,16 @@ public sealed class BingConnector : IWebSearchEngineConnector
     private readonly ILogger _logger;
     private readonly HttpClient _httpClient;
     private readonly string? _apiKey;
+    private readonly string? _uri;
+    private readonly string defaultUri = "https://api.bing.microsoft.com/v7.0/search?q";
 
     /// <summary>
     /// Initializes a new instance of the <see cref="BingConnector"/> class.
     /// </summary>
     /// <param name="apiKey">The API key to authenticate the connector.</param>
     /// <param name="logger">An optional logger to log connector-related information.</param>
-    public BingConnector(string apiKey, ILogger<BingConnector>? logger = null) :
-        this(apiKey, new HttpClient(NonDisposableHttpClientHandler.Instance, false), logger)
+    public BingConnector(string apiKey, string uri = "", ILogger<BingConnector>? logger = null) :
```

Comment:
If we mark `defaultUri` as constant, we can then use it here as default value - `string uri = DefaultUri`. In this case it will be visible for class consumers which URI will be used by default:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/4f25dfad-9c4e-4599-a05d-b288b9d367e8)

Same applies for constructor below.

Reply:
Agreed, went with your suggestion.